ansible-ipi-install/roles/node-prep/tasks/60_enabled_services.yml

```diff
@@ -6,8 +6,16 @@
     enabled: yes
   become: yes
   with_items:
-  - "{{ firewall }}"
   - libvirtd
   tags: services
 
-
+- name: Enable Services (iptables)
+  service:
+    name: "{{ item }}"
+    state: restarted
+    enabled: yes
+  become: yes
+  with_items:
+  - "{{ firewall }}"
```

Comment:
don't we also need libvirtd here?

Reply:
For libvirt is done in the above service, what I did is to move firewall to its own stanza so that we can use the 'when'

Reply:
I included firewalld here because I actually had someone open a bug that firewalld was not installed by default in their env. Essentially I ensure it is for either both firewalld or iptables

Reply:
I've created another stanza to ensure that firewalld is started (but not restarted)